### ISSUE TITLE
One minute between these 2 blocks

### ISSUE THREAD

**Issue by botstorm** [CREDITSCOM/node#1]
we finished test and chain is slow. here is some log with no transactions over network 
[2019-05-15 23:33:37.023910] [0x00007f14af928700] [debug] Normal: nothing to do with transactions packet in this state
[2019-05-15 23:33:37.023925] [0x00007f14af928700] [debug] TRANSPORT> POSTPHONED finished, round 2398489
[2019-05-15 23:34:22.480128] [0x00007f14af928700] [debug] TRANSPORT> Got SS Last Block: 2398488
[2019-05-15 23:34:26.502397] [0x00007f14af928700] [warning] -----------------------------------------------------------
[2019-05-15 23:34:26.502491] [0x00007f14af928700] [warning] NODE> BigBang #2398489: last written #2398488, current #2398489
[2019-05-15 23:34:26.502546] [0x00007f14af928700] [warning] -----------------------------------------------------------

here one minute between these 2 blocks

**Comment by kondrashovsv** [comment]
I also have

**Comment by micmac0** [comment]
> I also have
> ![image](https://user-images.githubusercontent.com/41994386/58038194-fbd57980-7b37-11e9-9817-f8f6d7131ecc.png)

About this graph here are test conditions we had :
3 nodes configured receive each less than 300tps.  (delay set to 3ms between each trx and we need to count connexion cost from client to node)
That s why we clearly see some level when we added removed nodes.
Around 8h20 we started to see some delay. When some nodes in network can t manage to deal with too much load, blocks start to have round number of transactions such as 800 1000 2000 4000 or number like that. When all is ok number are much more random like 478 832 ...
